index.src.html

```diff
@@ -4205,6 +4212,14 @@
             [=combined buffer layout/allocationSize=].
     9. Return |combinedLayout|.
 
+: <dfn>Convert PredefinedColorSpace to VideoColorSpace</dfn> (with |colorSpace|)
+ :: 1. This algorithm <em class="rfc2119">MUST</em> be called only if |colorSpace|
+        is equal to one of {{srgb}}, {{display-p3}}.
```

Comment:
Since this is a requirement on specifications rather than implementations, use https://infra.spec.whatwg.org/#assertions:

```suggestion
 :: 1. Assert: |colorSpace| is equal to one of {{srgb}} or {{display-p3}}.
```